### ISSUE TITLE
Suboptimal display of multiple-arearef callouts

### ISSUE THREAD

**Issue by sknorr** [openSUSE/suse-xsl#317]
See `tests/dapscompare-tests/db5-command` -> final callout "brunch tumeric"

The HTML behavior sucks but is tolerable: callout bugs shown below each other.

The PDF behavior plain doesn't look okay: callout bugs are shown next to each other but are overlaid by the associated para's text.

I am not quite sure what the best fix is here:

1. make the left column wider
2. add a line break before the para if there are multiple line callout bugs that need to be shown (not sure how to realize that one in FO, though.

reported by @tbazant

**Comment by tbazant** [comment]
or maybe do a firsts-line indent based on the number of joined calloutrefs?

**Comment by sknorr** [comment]
Right, that is option 3 -- might even look best but I am unsure how to make it happen either.

**Comment by cwickert** [comment]
I agree option 3 is probably what we want. I'm facing this problem in doc-caasp now, where we have yaml files with values that need to be replaced. Some show up in the file once, others twice or thrice. It's hard to align that properly, so we just indent the line depending on the number of callouts.

Any idea if/when we can get this fixed? Is this something @tomschr should look at?

**Comment by tomschr** [comment]
Tested with Jana in HTML. This is still valid.
